sdks/python/apache_beam/runners/runner.py

```diff
@@ -96,7 +96,7 @@ def create_runner(runner_name):
   if '.' in runner_name:
     module, runner = runner_name.rsplit('.', 1)
     try:
-      return getattr(__import__(module, {}, {}, [runner], -1), runner)()
+      return getattr(__import__(module, {}, {}, [runner], 0), runner)()
```

Comment:
A note for other reviewers from `__import__` documentation:

> level specifies whether to use absolute or relative imports. The default is -1 which indicates both absolute and relative imports will be attempted. 0 means only perform absolute imports.

Reply:
Could you also explain why is this change needed?

Reply:
I think the reason is the same as why we add `from __future__ import absolute_import` elsewhere - to explicitly disable implicit intra-package import behavior in Python 2 for forward-compatibility with Python 3. @RobbeSneyders, were some tests affected by this?

Related:
http://python-future.org/imports.html
https://www.python.org/dev/peps/pep-0302/

Reply:
In Python 3.3+, the -1 option is not available anymore.
Looking at the [_RUNNER_MAP](https://github.com/apache/beam/blob/363d3e385bdb9aca9f194a2b772038bc5db3cb5b/sdks/python/apache_beam/runners/runner.py#L40), all defined runner paths are absolute anyway.